.gitignore

```diff
@@ -32,3 +32,12 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
Here there's no empty line at the end of the file. 

You may want to configure Prettier to do this for you in a [pre-commit hook](https://prettier.io/docs/en/precommit.html) or similar. 🤔 

This is not too important, just wanted to point it out!

Reply:
Thanks for the tip! I’ll look into it :)